nodejs/spa/src/main/webapp/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Keycloak Single-Page Application Example</title>
+    <script type="text/javascript" src="KC_URL/js/keycloak.js"></script>
+    <script type="text/javascript">
+        function output(content) {
```

Comment:
Instead of re-querying the DOM every time I would recommend to store  the elements in variables.
```js
const outputElement = document.getElementById('output');
const nameElement = document.getElementById('name');
const userElement = document.getElementById('user');
```

Reply:
Done